CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Change Log
 
+## [vX.X.X](https://github.com/eclipse-iceoryx/iceoryx/tree/v1.0.0) (unreleased)
+
+[Full Changelog](https://github.com/eclipse-iceoryx/iceoryx/compare/v1.0.1...master)
+
+**Features:**
+- Enhance posixCall[\#805](https://github.com/eclipse-iceoryx/iceoryx/issues/805)
```

Comment:
This one would qualify as breaking change. Ideally there would also be a small code example how to port to the new API